### PR TITLE
[13.0][MIG] account_fiscal_position_vat_check

### PR DESCRIPTION
**Pull request by Garcicasti**

### CODE REVIEW

**Review by juliap98 — APPROVED** [review]
Funcional review ok

File: account_fiscal_position_vat_check/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2019 Akretion France (https://akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
```
Comment (ValentinVinagre) [inline account_fiscal_position_vat_check/models/account_invoice.py:10]: This model doesn't exist in v.13 now is "account.move"

File: account_fiscal_position_vat_check/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    @api.multi
+    def action_move_create(self):
```
Comment (ValentinVinagre) [inline account_fiscal_position_vat_check/models/account_invoice.py:13]: This function doesn't exist in v.13

File: account_fiscal_position_vat_check/models/account_move.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def action_move_create(self):
```
Comment (ValentinVinagre) [inline account_fiscal_position_vat_check/models/account_move.py:12]: the same, doesn't exist in v.13

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]
Please review the changes into versions.

**Comment by misern2** [comment]
@Garcicasti please could you attend the comments.

Thanks!

**Pull request closed without merging**